webpack/JobWizard/steps/HostsAndInputs/buildHostQuery.js

```diff
@@ -4,7 +4,7 @@ export const buildHostQuery = (selected, search) => {
   const hostCollectionsSearch = `(host_collection_id ^ (${hostCollections
     .map(({ id }) => id)
     .join(',')}))`;
-  const hostGroupsSearch = `(hostgroup_id ^ (${hostGroups
+  const hostGroupsSearch = `(hostgroup ^ (${hostGroups
     .map(({ id }) => id)
```

Comment:
Actually, the more I look in this file the more I want to change few things :/
 - If `hostGroups`/`hosts`/`hostCollections` contains both `id` and `name`, why we use `id` here although we're searching by name? Moreover, why `id` contains the same value as `name`?
 - If we're going to use names, all the names should be quoted in case they have spaces, e.g. `Katello CentOS 7`, so it should be 
```suggestion
  const hostGroupsSearch = `(hostgroup ^ (${hostGroups
    .map(({ id }) => `"${id}"`)
```

And I guess initially it supposed to use ids instead of names, so here `host*_id` are being used. The better fix for this helper will be to ensure `selected` contains actual `id` value and then use this value to avoid dealing with the strings/spaces/quotes.

UPD: I noticed that for host collections it works as expected, it uses actual ids.

Reply:
I'm afraid hostgroup names can contain quotes so unless we escaped the name, just wrapping it in quotes still wouldn't be perfect. I'd rather go the id route